app/components/curriculum_video_component/curriculum_video_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="video-component-<%= layout_class %>">
+  <div class="video-component-<%= layout_class %>__title">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Career video to support the unit</h2>
+  </div>
+  <div class="video-component-<%= layout_class %>-content">
+    <div class="video-component-<%= layout_class %>-content__media">
+      <%= render YouTubeEmbedComponent.new(video_url: @video_url) %>
+    </div>
+    <div class="video-component-<%= layout_class %>-content--details">
```

Comment:
I wouldn't use the double dash between content and details, as -- implies details is a modifier, where I think you mean its a element, so should be double underscore?

Or as it contains other elements, it might just be best keeping it a - but only a single one, so its not confused as a modifier

Reply:
Agreed. I'll change this to a single dash.